backend/layers/common/entities.py

```diff
@@ -80,6 +80,7 @@ class DatasetArtifactType(str, Enum):
     H5AD = "h5ad"
     RDS = "rds"
     CXG = "cxg"
+    ATAC_FRAGMENT = "bgz"  # TODO: This is not great, extension doesn't exactly map to "artifact type"
```

Comment:
@Bento007 I don't love this, but AFAICT this is what the value *should* be.

WDYT?

Reply:
I think having another table to translate the identifier we use and the actual extension would be helpful

Reply:
That makes sense, but I have two concerns:

* Making it so the DatasetArtifactType value is not used as the extension will touch a lot of code
* The only case where the value is different (AFAIK) doesn't actually exist in the system/ database yet so I don't think I'm going to find all the cases

My personal preference for implementation would be to just no longer use an Enum here, and instead use classes so we can extend when needed. But I think that's a much larger code change, so probably won't go that direction.